utils/src/test/kotlin/ProcessCaptureTest.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.utils
+
+import io.kotlintest.shouldBe
+import io.kotlintest.specs.StringSpec
+
+class ProcessCaptureTest : StringSpec({
+    "Environment variables should be passed correctly" {
+        val env = mapOf("GOPATH" to "/foo/bar")
```

Comment:
`GOPATH` is a very specific name without context here, what about using a more generic name like `SOME_PATH` or just `VARIABLE`? And since the test name contains "variable**s**", maybe add a second one?

Reply:
I had originally done this while debugging https://github.com/heremaps/oss-review-toolkit/issues/1233, hence the naming, but I'll rework it.